product-manual/intent-signals.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: "Intent Signals"
+description: "Learn more about Intent Signals"
+---
+
+Outline
+
+- what is an Intent Signal?
```

Comment:
Cross link to your guide? (Unclear if the product manual is the same as your guide)